DeliBuddy/app/src/main/java/yapp/android1/delibuddy/model/Address.kt

```diff
@@ -0,0 +1,28 @@
+package yapp.android1.delibuddy.model
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+import yapp.android1.domain.entity.AddressEntity
+
+@Parcelize
+data class Address(
+    val addressName: String,
+    val roadAddress: String,
+    val address: String,
+    var addressDetail: String,
+    val lat: Double,
+    val lng: Double
+) : Parcelable {
+    companion object {
```

Comment:
이 부분 보다가 생각난건데
MutableStateFlow에 기본값으로 Address를 넣어줄때
companion object에 DEFAULT Address를 미리 선언해주면 보기도 좋고 나중에 다시 쓸때도 편해요
```suggestion
    companion object {
        val DEFAULT = Address(
            addressName = "서울역",
            roadAddress = "서울특별시 중구 한강대로 405",
            address = "",
            addressDetail = "",
            lat = 37.5283169,
            lng = 126.9294254
        )
    }
```

Reply:
아 맞아요! 영상 님 코드 중에 이렇게 정의해 두고 사용하시는 걸 보고 고쳐보면 좋겠다 했는데 이참에 적용해서 고쳐보겠습니당